### PR TITLE
Restyle the website

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Now that we have a gallery of example, let's make the website a bit prettier.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -24,11 +24,8 @@
   </style>
 
 
-.. raw:: html
-
-    <h1 style='text-align: center; font-size: 270%;'>
-    Joblib: running Python functions as pipeline jobs
-    </h1>
+Joblib: running Python functions as pipeline jobs
+==================================================
```
Comment (aabadie) [inline doc/index.rst]: Nit: there is an extra `=` on this line

**Comment by aabadie** [comment]
For people interested, the new style can be seen [here](https://18-654608-gh.circle-artifacts.com/0/home/ubuntu/joblib/build/sphinx/html/index.html)

I'm not fully convinced by the new style, sorry. I like the actual black/orange theme which for me represents joblib's visual identity.

**Comment by lesteve** [comment]
I have to say I don't really have a strong feeling about joblib's visual identity (maybe we should wait until it has a logo before really caring?) and I am fine with using alabaster theme.

**Comment by GaelVaroquaux** [comment]
Alex liked it. Merging.

**Comment by GaelVaroquaux** [comment]
Things can be improved. This is a start

**Comment by lesteve** [comment]
Before I forget, it would be great if the Gallery of Examples was in the sections listed on the right section, to make it more accessible.

**Comment by aabadie** [comment]
> Alex liked it

I said I was *not* convinced but maybe this is another Alex ;)

**Pull request merged**